### ISSUE TITLE
Parameter and LookupParameter cannot be created from numpy array in python

### ISSUE THREAD

**Issue by zhechenyan** [clab/dynet#1299]
Following the Jupiter-tutorial/API.ipynb, I tired to create a Parameter from a numpy array initialized to 0s with the command 
`p = m.add_parameters(np.zeros((3,5)))`  but failed with the following error message.
```
---------------------------------------------------------------------------
AssertionError    Traceback (most recent call last)
<ipython-input-116-b6981ad6a79f> in <module>()
----> 1 p = m.add_parameters(np.zeros((3,5)))

_dynet.pyx in _dynet.ParameterCollection.add_parameters()

_dynet.pyx in _dynet.ParameterCollection.add_parameters()

AssertionError: First argument of add_parameters should be a valid dimension or a numpy array
```

The same issue when trying to create a LookupParameter. 
(`lp = m.add_lookup_parameters(np.zeros((3,5)))`) With the same error.

**Comment by zhechenyan** [comment]
an old API issue. Solved by adding the shape of the np array.
`p = m.add_parameters((3,5), np.zeros((3,5)))` can solve this problem.

**Comment by neubig** [comment]
This should be fixed in the process of fixing #1302
